### PR TITLE
v0.0.2

### PR DESCRIPTION
**Pull request by ewilz**
sooo....this has already been published, so I'm not sure I want to change stuff....but maybe comments from this PR could go into the next version?? lmk

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,10 +1,14 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "lib": ["ES6"],
+    "lib": ["dom", "ES6"],
```
Comment (ewilz) [inline tsconfig.json:4]: I actually don't think this is needed, will probably get rid of this for v0.0.3

**Comment by ewilz** [comment]
People who I can think of that may want to review is @nambrot and @eelanagaraj

File: tsconfig.json (hunk 1 above)
Comment (eelanagaraj) [reply tsconfig.json:4]: ooh yeah I think you're right with the addition of the `skipLibCheck`!

**Review by eelanagaraj — APPROVED** [review]
LGTM!

**Pull request merged**